src/components/pages/campaigns.js

```diff
@@ -58,7 +58,8 @@ function Campaigns() {
       </Helmet>
       <Stack
         direction={{ base: 'column', md: 'row' }}
-        spacing={2}
+        spacing={5}
+        margin="2rem"
```

Comment:
eğer `spacing` kullanıyorsak tekrardan margin yazmaya gerek yok. `stack` component'ının `spacing` prop'u boşluk bırakmayı sağlıyor. daha fazla detay için [chakra-ui stack component](https://chakra-ui.com/docs/layout/stack) dokümantasyonuna bakabilirsin.